packages/router/README.md

```diff
@@ -8,7 +8,7 @@ Redwood Router (RR from now on) is designed to list all routes in a single file,
 
 ## Installation
```

Comment:
I think this entire section can be removed now. I looked at our other README files, and they don't have installation sections.

What do you think?

Reply:
Agreed, if you can only use it in a Redwood app, it's installed when you create the Redwood app. No need for this section at all.

Reply:
LGTM